MANIFEST.in

```diff
@@ -1,7 +1,6 @@
 include setup*.py
 include MANIFEST.in
-include README.md
-include LICENSE.txt
+include *.md
```

Comment:
Why this change?

Reply:
Hum, I think I or my IDE did something weird.